product_docs/docs/pgd_cli/1/rel_notes/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "PGD CLI Release notes"
+navTitle: Release notes
+---
+
+PGD CLI is a tool to manage your BDR cluster. It allows you to run commands against BDR clusters. 
+
+The PGD CLI documentation describes the latest version of PGD CLI including minor releases and patches. The release notes in this section provide information on what was new in each release.
+
+
+| Version                  | Release Date | 
+| -----------------------  | ------------ | 
+| [1](epd_cli_1_rel_notes) | 2022 May 17  |
```

Comment:
I think we should name the file as `pgd_cli_1_rel_notes` instead of `epd_cli_1_rel_notes`